ledger-api/grpc-definitions/com/daml/ledger/api/v1/completion.proto

```diff
@@ -20,8 +20,7 @@ message Completion {
   string command_id = 1;
 
   // Identifies the exact type of the error.
-  // For example, malformed or double spend transactions will result in a ``INVALID_ARGUMENT`` status.
-  // Transactions with invalid time time windows (which may be valid at a later date) will result in an ``ABORTED`` error.
+  // The error is reported using the same rules for error codes and messages as the errors returned for API requests.
```

Comment:
```suggestion
  //  Uses the same format of conveying error details as is used for the LAPI rpcs.
```
?

Reply:
There is currently zero occurrences of "LAPI" in the proto files. how about

```suggestion
  // Uses the same format of conveying error details as is used for the RPC responses of APIs.
```

Reply:
Yes, good point. I would also use a pronoun at the beginning of the sentence: "It uses..."

Reply:
```suggestion
  // It uses the same format of conveying error details as it is used for the RPC responses of the APIs.
```